cvat-ui/src/actions/cloud-storage-actions.ts

```diff
@@ -103,40 +103,16 @@ export type CloudStorageActions = ActionUnion<typeof cloudStoragesActions>;
 
 export function getCloudStoragesAsync(query: Partial<CloudStoragesQuery>): ThunkAction {
     return async (dispatch: ActionCreator<Dispatch>): Promise<void> => {
-        function camelToSnake(str: string): string {
-            return (
-                str[0].toLowerCase() + str.slice(1, str.length)
-                    .replace(/[A-Z]/g, (letter) => `_${letter.toLowerCase()}`)
-            );
-        }
-
         dispatch(cloudStoragesActions.getCloudStorages());
         dispatch(cloudStoragesActions.updateCloudStoragesGettingQuery(query));
 
         const filteredQuery = { ...query };
         for (const key in filteredQuery) {
-            if (filteredQuery[key] === null) {
-                delete filteredQuery[key];
+            if ((filteredQuery as { [index: string]: any })[key] === null) {
+                delete (filteredQuery as { [index: string]: any })[key];
```

Comment:
Maybe declare interface for `{ [index: string]: any }`? I see there are the repetitions of this casting in other files

Reply:
Makes sense

Reply:
Fixed